Sources/ConsoleKit/Utilities/ConsoleLogger.swift

```diff
@@ -1,5 +1,114 @@
 import Logging
 
+/// splits a path on the /Sources/ folder, returning everything after
+///
+///     "/Users/developer/dev/MyApp/Sources/Run/main.swift"
+///     // becomes
+///     "Run/main.swift"
+///
+func conciseSourcePath(_ path: String) -> String {
+#if compiler(>=5.3)
+	return path
+#else
+	let separator: Substring = path.contains("Sources") ? "Sources" : "Tests"
+	return path.split(separator: "/")
+		.split(separator: separator)
+		.last?
+		.joined(separator: "/") ?? path
+#endif
+}
```

Comment:
The minimum Swift version for the package is now 5.6; you don't need this utility method at all 😅 
```suggestion
```